src/qtwebengine/dataprovider.h

```diff
@@ -1,14 +1,36 @@
-#ifndef DATAPROVIDER_QWEBENGINE_H
-#define DATAPROVIDER_QWEBENGINE_H
+/*
+ *  Kchmviewer - a CHM and EPUB file viewer with broad language support
+ *  Copyright (C) 2004-2016 George Yunaev, gyunaev@ulduzsoft.com
+ *  Copyright (C) 2021 Nick Egorrov, nicegorov@yandex.ru
```

Comment:
This is rarely included in pull requests, honestly. Especially for minor changes.